docs/framework/Release-Notes/01-January-Security-and-Quality-Rollup

```diff
@@ -0,0 +1,298 @@
+<h2>January Security and Quality Rollup</h2>
```

Comment:
Can we make the preview post after the security and quality post in the same month so that the posts are in the release sequence?